service/Makefile

```diff
@@ -1,9 +1,6 @@
 backup-db:
 	docker exec -i $(shell docker ps -qf label=nl.surfcatalog.db) pg_dump -h localhost -U postgres -c edushare > edushare.${now}.postgres.sql
 
-import-db:
```

Comment:
Hier staat nog iets over in de README, kun je die ook aanpassen?

Reply:
Yes gedaan, zit een kleine refactor in om van S3 downloaden te delen tussen invoke (local) en fabric (ssh) commands.